src/algokit/cli/tasks/mint.py

```diff
@@ -211,7 +211,10 @@ def mint(  # noqa: PLR0913
 
     client = load_algod_client(network)
     validate_balance(
-        client, creator_account, 0, algos_to_microalgos(ASSET_MINTING_MBR)  # type: ignore[no-untyped-call]
+        client,
```

Comment:
Is this a formatting change? Ideally we want to avoid formatting differences to reduce churn on source files as they get edited.

Reply:
Ruff claims that Black-formatted projects like Django and Zulip, > 99.9% of lines are formatted identically and non-Black-formatted code, the formatter makes some different decisions than Black.
So either way there will be little changes. I think this is one of the 0.01% cases.

Reply:
https://docs.astral.sh/ruff/formatter/black/

Reply:
Cool, happy!